### PR TITLE
Re-enable level 10 conformance test

### PR DESCRIPTION
**Pull request by eustas**

### CODE REVIEW

**Review by veluca93 — APPROVED** [review]

**Review by mo271 — DISMISSED** [review]

**Comment by eustas** [comment]
Ooops.
```
Testing alpha_premultiplied
Traceback (most recent call last):
  File "conformance/scripts/conformance.py", line 219, in <module>
    main()
  File "conformance/scripts/conformance.py", line 214, in main
    if not ConformanceTestRunner(args):
  File "conformance/scripts/conformance.py", line 178, in ConformanceTestRunner
    CompareNPY(reference_npy, reference_icc, decoded_npy,
  File "conformance/scripts/conformance.py", line 48, in CompareNPY
    raise ConformanceTestError(
__main__.ConformanceTestError: RMSE too large: 0.000277058[13](https://github.com/libjxl/libjxl/runs/5661597831?check_suite_focus=true#step:6:13)40882927 > 3.8[15](https://github.com/libjxl/libjxl/runs/5661597831?check_suite_focus=true#step:6:15)e-06
```

File: .github/workflows/conformance.yml [hunk 1]
```diff
@@ -108,7 +107,7 @@ jobs:
       uses: actions/checkout@v2
       with:
         repository: libjxl/conformance
-        ref: ce5a804a3ff834097b93c39d2f93e5445f6bf340
```
Comment (veluca93) [inline .github/workflows/conformance.yml:111]: This is actually the correct hash :) `master` on `libjxl/conformance` is not the most recent/up to date conformance test.

Fixing that...

**Comment by eustas** [comment]
@veluca93 PTAL

**Comment by eustas** [comment]
@veluca93 please rubber-stamp

**Review by veluca93 — APPROVED** [review]

**Pull request merged**